message_definitions/v1.0/ras_a.xml

```diff
@@ -10,7 +10,140 @@
+        <wip/>
+        <!-- This command is work-in-progress and it can therefore change. It should NOT be used in stable production environments. -->
+        <description>
+          Set exploration task world boundaries. When starting a new behavior, either the system has default boundaries or it iss boundless.
+          This message will be accepted when the task ID already exists, or otherwise should fail. In order to set the exploration task world boundaries,
+          p1 and p2 identify the vertices of a rectangle, which define a 2D localization of the exploration area, while p3, matching the same coordinates of p1,
+          provides the height limitation of the 3D volume to explore.
+          In short, the coordinates are x1, y1, x2, y2, z and the height of the cuboid / 3D volume, where the point coordinates can be represented by
+          p1(x1, y1, z+h), p2(x2, y2, z+h) and p3(x1, y1, z). COMMAND_INT should be used so to set the MAV_FRAME and consequently,frame and coordinates of the boundaries
+          cuboid (MAV_FRAME_GLOBAL means global coordinates are being used, while MAV_FRAME_LOCAL_NED means local NED coordinates are being used).
+        </description>
+        <param index="1" label="Exploration task ID" minValue="0" maxValue="255" increment="1">ID of the exploration task to set these boundaries to. Set to UINT8_MAX if not applicable or/and to set these boundaries to the current running or active exploration task.</param>
+        <param index="2" label="Cuboid height" units="m">Exploration 3D space boundaries cuboid height. The Z local coordinate or altitude (MSL) of point 1 and 2 are computed by the sum of this height with the local Z or altitude (MSL) of point 3, i.e Z1 = Z2 = Z3 + cuboid height. NaN if not applicable.</param>
+        <param index="3" label="Approach vector initial point Y or Longitude">Local X (mE4) or Latitude (degE7) (WGS84) of point 1 of the exploration 3D space boundaries cuboid. NaN or INT32_MAX if not applicable.</param>
+        <param index="4" label="Approach vector initial point Z or Altitude (MSL)">Local Y (mE4) or Longitude (degE7) (WGS84) of point 1 of the exploration 3D space boundaries cuboid. NaN or INT32_MAX if not applicable.</param>
+        <param index="5" label="Approach vector final point X or Latitude">Local X (mE4) or Latitude (degE7) (WGS84) of point 2 of the exploration 3D space boundaries cuboid. NaN or INT32_MAX if not applicable.</param>
+        <param index="6" label="Approach vector final point Y or Longitude">Local Y (mE4) or Longitude (degE7) (WGS84) of point 2 of the exploration 3D space boundaries cuboid. NaN or INT32_MAX if not applicable.</param>
+        <param index="7" label="Approach vector final point Z or Altitude (MSL)" units="m">Local Z or Altitude (MSL) point 3 of the exploration 3D space boundaries cuboid. This also represents the height of the bottom plane of the cuboid. NaN if not applicable.</param>
```

Comment:
"Approach vector" doesn't seem applicable, nor do "initial" and "final". Param 4's label and description don't match.